src/applications/personalization/profile/tests/components/ContactInformationField.unit.spec.jsx

```diff
@@ -81,34 +81,6 @@ describe('<ContactInformationField/>', () => {
 
     component.unmount();
   });
-
```

Comment:
This whole file should be deleted IMO. The tests are not very valuable and everything the tests try to cover are things that other tests cover better.